packages/server/lib/request.coffee

```diff
@@ -132,7 +133,13 @@ createCookieString = (c) ->
 module.exports = (options = {}) ->
   defaults = {
     timeout: options.timeout ? 20000
-    # forever: true
+    agent: agent
```

Comment:
is this being tested somewhere so that our own `lib/api` code that interacts with the dasbhoard goes through here?

Reply:
looks like `lib/api` just uses `request-promise` directly, would it be better to use `request.coffee` or just use the Agent in `lib/api`?

Reply:
I changed `lib/api` to use the agent directly, since `request.coffee` seems to be more for user-made requests

Reply:
yup agreed